spec/mon_spec.rb

```diff
@@ -6,7 +6,7 @@
   ALLPLATFORMS.each do |pltfrm|
     context "on #{pltfrm[:platform]} #{pltfrm[:version]}" do
       cached(:chef_run) do
-        ChefSpec::SoloRunner.new(pltfrm).converge(described_recipe)
+        ChefSpec::SoloRunner.new(pltfrm.dup.merge(step_into: %w(munin_plugin))).converge(described_recipe)
```

Comment:
We shouldn't be stepping into the ``munin_plugin`` resource and instead should only be checking the munin_plugin resource in ChefSpec.